tools/vscode/refresh_compdb.sh

```diff
@@ -2,10 +2,7 @@
 
 [[ -z "${SKIP_PROTO_FORMAT}" ]] && tools/proto_format/proto_format.sh fix
 
-bazel_or_isk=bazelisk
-command -v bazelisk &> /dev/null || bazel_or_isk=bazel
-
-opts=(--vscode --bazel="$bazel_or_isk")
```

Comment:
Removed `bazelisk` support because:

1. we use `bazel` command directly at almost everywhere in our scripts, like at `ci/do_ci.sh`.
2. our `bazel/README.md` has recommended to install `bazelisk` as `bazel` command.
3. aliases of binnary commands should be handled by callers/users self.

Tools of C++ is complex enough, let's keep our script as simple as possible.

Reply:
And I am thinking if we can remove the `refresh_compdb.sh` directly and make it an function in the `ci/do_ci.sh` because it only contains few lines code that will call the `gen_compilation_database.py`.

cc @phlax

Reply:
removing the bazel_or_isk logic sgtm

less sure about removing the script as it seems to be doing a couple of other things also - given this is apparently for vscode i think we want to ensure we keep the existing code path (ie without do_ci.sh) unless we are 100% confident that it wont break anything vscode

Reply:
> less sure about removing the script as it seems to be doing a couple of other things also - given this is apparently for vscode i think we want to ensure we keep the existing code path (ie without do_ci.sh) unless we are 100% confident that it wont break anything vscode

I use this script every day, haha. But I see your worry. I will try to see if we can remove the minor vscode things from the script first.

Reply:
The only vscode related thing is `--vscode` parameter. It will replace the `-iquote` with old style `-I`. It mainly is used to make the c++ intellisense works. But the clangd is our recommend c++ extension rather than the c++ intellisense.

see https://github.com/envoyproxy/envoy/pull/5873

Reply:
Ok, the special support to the c++ intellisense is still necessary. But maybe in some other way. Let me create a new PR this weekend to do this.